### PR TITLE
CUMULUS-2889 update CloudwatchLogs code to sdk v3

### PR DESCRIPTION
**Pull request by Nnaga1**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by jennyhliu** [comment]
I suggest to add a couple of unit tests (such as createLogGroup)to verify the cloudwatchlogs service is working

**Review by etcart — APPROVED** [review]

File: packages/aws-client/tests/test-CloudWatchLogs.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const result = await cloudwatchlogs().createLogGroup({ logGroupName });
+  t.is(result.$metadata.httpStatusCode, 200);
+  const description = await cloudwatchlogs().describeLogGroups({
+    logGroupNamePattern: logGroupName,
+  });
+  t.is(description.logGroups.length, 1);
+  t.is(description.logGroups[0].logGroupName, logGroupName);
+  await cloudwatchlogs().deleteLogGroup({ logGroupName });
+});
+
```
Comment (jennyhliu) [inline packages/aws-client/tests/test-CloudWatchLogs.js:18]: The two tests seem the same, can we combine?

File: tf-modules/archive/api.tf [hunk 2]
```diff
@@ -149,7 +149,7 @@ resource "aws_lambda_function" "private_api" {
   handler          = "index.handler"
   role             = aws_iam_role.lambda_api_gateway.arn
   runtime          = "nodejs16.x"
-  timeout          = lookup(var.lambda_timeouts, "PrivateApiLambda", 100)
+  timeout          = lookup(var.lambda_timeouts, "PrivateApiLambda", 900)
```
Comment (Jkovarik) [inline tf-modules/archive/api.tf:152]: What was the reason for this update?    We should be *very* careful about upping this timeout.

**Review by Jkovarik — DISMISSED** [review]

File: example/cumulus-tf/lambdas.tf [hunk 3]
```diff
@@ -148,6 +148,7 @@ resource "aws_lambda_function" "ftpPopulateTestLambda" {
   role             = module.cumulus.lambda_processing_role_arn
   runtime          = "nodejs16.x"
   timeout          = 150
+  memory_size      = 256
```
[the remark this answers is not in the document]
Comment (etcart) [reply example/cumulus-tf/lambdas.tf:151]: yea, the reason for this is that these integration tests were failing due to memory overflow.
but if we feel these should be in the changelog, fair enough. I don't believe this function is being used in production, but only for our tests

[the remark this answers is not in the document]
Comment (Jkovarik) [reply example/cumulus-tf/lambdas.tf:151]: It's reasonable to log example project changes - tests *are* production code.      We just need to be sure it's clear what we're talking about.

**Pull request closed without merging**